test/package.js

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs';
+import test from 'ava';
+import pify from 'pify';
+import Promise from 'pinkie-promise';
```

Comment:
You don't need this. `global.Promise = Promise;` works fine since Babel replaces the right part with a polyfill.